ntbs-service/wwwroot/source/Components/ValidateRequiredCheckboxes.ts

```diff
@@ -0,0 +1,49 @@
+import Vue from 'vue';
+import { getHeaders, getValidationPath } from '../helpers';
+const axios = require('axios');
+const qs = require('qs');

```

Comment:
We should try and keep imports consistent in style. Does it work fine if we replace with
`import qs from 'qs';`?
(and ditto for axios)

Reply:
unfortunately qs doesn't work with import, but axios did